src/Spice86/ViewModels/MainWindowViewModel.cs

```diff
@@ -395,6 +395,18 @@ await MessageBox.Avalonia.MessageBoxManager.GetMessageBoxStandardWindow("", "Ple
             _isSettingResolution = false;
         }, DispatcherPriority.MaxValue);
     }
+    
+    public void SetResolution(int width, int height) {
+        Dispatcher.UIThread.Post(() => {
+            _isSettingResolution = true;
+            DisposeBuffers();
+            VideoBuffers = new();
+            Width = width;
+            Height = height;
+            AddBuffer(_videoCard, 0xA0000, 1, width, height, true);
```

Comment:
Is the usage of 0xA0000 here work with video modes that don't start at that address ?

Reply:
Actually, it completely ignores that address. Also in the existing code.

I'd have to think a bit on how to make it so it can take an arbitrary address rather than the one programmed into the registers.

Reply:
Removing the feature of adding/remove arbitrary video buffers from the UI and IGui is fine too.